### PR TITLE
net-irc/emech: Version bump. Bump EAPI, fix building

### PR DESCRIPTION
**Pull request by cla86**

### CODE REVIEW

**Comment by cla86** [comment]
No opened bugs.

File: net-irc/emech/emech-3.0.99_p3.ebuild [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils readme.gentoo-r1 toolchain-funcs
```
Comment (Zlogene) [inline net-irc/emech/emech-3.0.99_p3.ebuild]: do you really need `eutils` here with EAPI=6 now?

File: net-irc/emech/emech-3.0.99_p3.ebuild [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+IUSE="debug session tcl"
+S="${WORKDIR}/${P/_/}"
+
+DOC_CONTENTS="You can find a compressed sample config file at /usr/share/doc/${PF}"
+
+src_prepare() {
+	epatch "${FILESDIR}/${P}-buildfix.patch"
```
Comment (Zlogene) [inline net-irc/emech/emech-3.0.99_p3.ebuild]: no `eaptch` in EAPI=6 but `eapply`

Comment (cla86) [reply net-irc/emech/emech-3.0.99_p3.ebuild]: Fixed in 4f44d34

**Comment by mgorny** [comment]
Since this package has no maintainer, would you be interested in maintaining it? (i.e. having the bugs reported to it assigned to you, and bumping it whenever appropriate) If you are, please add yourself and proxy-maint to `metadata.xml` (you can find examples in other ebuilds and on wiki).

**Comment by mgorny** [comment]
Please rebase this.

**Pull request closed without merging**